apps/admin-gui/src/app/vos/pages/group-detail-page/group-members/group-members.component.ts

```diff
@@ -99,6 +108,15 @@ export class GroupMembersComponent implements OnInit {
           }
           this.loading = false;
         }, err => this.loading = false);
+
+        this.attributesManager.getVoAttributeByName(voId, "urn:perun:vo:attribute-def:def:blockManualMemberAdding").subscribe(attrValue => {
+          this.blockManualMemberAdding = attrValue.value !== null;
+          if (this.blockManualMemberAdding !== true) {
+            this.attributesManager.getGroupAttributeByName(groupId, "urn:perun:group:attribute-def:def:blockManualMemberAdding").subscribe(groupAttrValue => {
+              this.blockManualMemberAdding = groupAttrValue.value !== null;
+            });
+          }
+        });
```

Comment:
please move this block of code "up"
we want to load the page when all the asynchronous operations are done

Reply:
This block of code has been moved.